pkgs/by-name/ag/agnos/package.nix

```
@@ -0,0 +1,35 @@
+{
+  fetchFromGitHub,
+  lib,
+  nixosTests,
+  rustPlatform,
+  stdenv,
+  darwin,
+  openssl,
+  pkg-config,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "agnos";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "krtab";
+    repo = "agnos";
+    rev = "v${version}";
+    hash = "sha256-hSiJvpTQIbhz/0AFBTvgfRDTqOi9YcDOvln15SksMJs=";
+  };
+
+  cargoHash = "sha256-PuiFzw7TGsTqzLvK5PV8jGBTzvkdh2BKTjeLMOuoUIE=";
+
+  buildInputs = [ openssl ] ++ lib.optional stdenv.isDarwin darwin.apple_sdk.frameworks.Security;
```

```suggestion
  buildInputs = [ openssl ];
```
I suspect this is no longer required since the SDK rework.